.kitchen.yml

```diff
@@ -12,10 +12,12 @@ platforms:
 suites:
   - name: default
     run_list:
-      - recipe[opsmatic::agent]
+     - recipe[opsmatic::agent]
+      
     attributes:
       {
         "opsmatic": {
-          "integration_token": "4f15f0b3-6881-41e0-a8a7-c1d9e85c8018"
+          "integration_token": "<YOUR INTEGRATION TOKEN>",
```

Comment:
the integration token in .kitchen.yaml should just be the token for our automated tests org. it's the test token, it's fine.